package.json

```diff
@@ -8,6 +8,10 @@
     "build": "next build",
     "nodemon": "nodemon",
     "reminder": "node config/dev/reminders.js",
+    "refresh-materialized-views": "ts-node lib/etl/jobs/materialized_views_refresh.ts",
```

Comment:
Added the nice names to the package.json. This makes it easy to run from the command line for testing, and I thought it would work remotely, but it is less flexible. Not sure what you'd like to do, but I thought changing the commands in the cron setup would be ideal.